modules/auth/ldap/README.md

```diff
@@ -103,3 +103,21 @@ share the following fields:
       matching parameter will be substituted with the user's username.
     * Example: (&(objectClass=posixAccount)(cn=%s))
     * Example: (&(objectClass=posixAccount)(uid=%s))
+
+**Verify group membership in LDAP** uses the following fields:
+
+* Group Search Base (optional)
+    * The LDAP DN used for groups.
+    * Example: ou=group,dc=mydomain,dc=com
+
+* Group Name Filter (optional)
+    * An LDAP filter declaring how to find valid groups in the above DN.
+    * Example: (|(cn=gogs_users)(cn=admins))
```

Comment:
```suggestion
    * Example: (|(cn=gitea_users)(cn=admins))
```

Reply:
Ack, changed these in the templates and forgot this one. :sweat_smile: